src/main/app/controller/PostController.ts

```diff
@@ -37,6 +37,8 @@ export class PostController<T extends AnyObject> {
       await this.saveAndSignOut(req, res, stepData);
     } else if (req.body.saveBeforeSessionTimeout) {
       await this.saveBeforeSessionTimeout(req, res, stepData);
+    } else if (req.body.isApplicant2) {
+      await this.saveAndContinue(req, res, stepData, CITIZEN_APPLICANT2_UPDATE);
```

Comment:
The CITIZEN_UPDATE event is being passed as 'eventName', while this is just CITIZEN_APPLICANT2_UPDATE
  Could this be more consistent maybe?

Reply:
Not sure what you mean?

Reply:
On line 43:
      await this.saveAndContinue(req, res, stepData, eventName);
      
Where 'eventName' is CITIZEN_UPDATE. 
Just think this could be a bit confusing